aptos-move/aptos-gas-schedule-updator/src/lib.rs

```diff
@@ -86,6 +86,11 @@ fn generate_script(gas_schedule: &GasScheduleV2) -> Result<String> {
         "gas_schedule::set_gas_schedule(&framework_signer, gas_schedule_blob);"
     );
 
+    emitln!(
+        writer,
+        "aptos_governance::reconfigure(&framework_signer);"
+    );
+
```

Comment:
In this approach `consensus_config::set` no longer calls `reconfigure`, I think the proposal script should call `reconfigure` explicitly. (Same below for some other configs)

@zekun000 @movekevin @sherry-x could you confirm if these release builder updates are also needed?